.kitchen.yml

```diff
@@ -1,21 +1,10 @@
 ---
-driver:
-  name: vagrant
-
 provisioner:
   name: chef_solo
   encrypted_data_bag_secret_key_path: test/integration/encrypted_data_bag_secret
   data_bags_path: test/integration/data_bags
 
-platforms:
-  - name: centos-6.6
-    run_list:
-      - recipe[python-webapp-test::_centos]
```

Comment:
This can be moved to the `default` suite since that recipe checks for the platform version. I will update the pr.